src/ui/preferences-window/tabs/ControlsTab.swift

```diff
@@ -5,11 +5,11 @@ class ControlsTab {
     static var shortcuts = [String: ATShortcut]()
     static var shortcutControls = [String: (CustomRecorderControl, String)]()
     static var shortcutsActions = [
-        "holdShortcut": { App.app.focusTarget() },
-        "holdShortcut2": { App.app.focusTarget() },
-        "holdShortcut3": { App.app.focusTarget() },
-        "holdShortcut4": { App.app.focusTarget() },
-        "holdShortcut5": { App.app.focusTarget() },
+        "holdShortcut": { App.app.handleHold(0) },
```

Comment:
I'm not sure about the name of this function, but I can't think of a better one :-)